src/styles/blocks/card.scss

```diff
@@ -0,0 +1,71 @@
+.card {
+  border: 1px solid $color-gray;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__description {
+    color: $text-main-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+  &__code {
+    color: $text-secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    color: $text-main-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__stars {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    margin-top: 16px;
+  }
```

Comment:
The use of both `margin-bottom` and `margin-top` is not efficient. Try to stick to a single direction for margins. For example, use only `margin-bottom` to create spacing between elements. This will make your code more consistent and easier to maintain.